modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/LineMBLayer.java

```diff
@@ -131,8 +174,8 @@ public Number getLineRoundLimit() {
      * 
      */
     public Number getLineOpacity() {
-        if (paintJson.get("line-opacity") != null) {
-            return (Number) paintJson.get("line-opacity");
+        if (paint.get("line-opacity") != null) {
+            return (Number) paint.get("line-opacity");
```

Comment:
Could not use that as it returns Expression, ended up doing it by hand. Thanks for the catch.